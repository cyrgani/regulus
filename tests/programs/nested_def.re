def(outer, x, _(
    def(inner, print("inner!")),
    def(inner2, print(x)),
    print("outer!"),
    inner2
)),

outer(2),
__builtin_print_catch(inner()),
=(inner2, outer(2)),
__builtin_print_catch(inner2()),
