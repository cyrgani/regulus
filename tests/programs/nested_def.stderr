NameError: No function `inner` found!
NameError: No variable named `x` found!
